samples/maps/demo/data-class-two-ranges/demo.js

```diff
@@ -14,24 +14,52 @@
         parsed: function (columns) {
 
             /**
-             * Event handler for clicking points. Use jQuery UI to pop up
-             * a pie chart showing the details for each state.
+             * Event handler for clicking points.
              */
             function pointClick() {
-                const row = this.options.row,
-                    $div = $('<div></div>')
-                        .dialog({
-                            title: this.name,
-                            width: 320,
-                            height: 300
-                        });
-
-                window.chart = new Highcharts.Chart({
+                const row = this.options.row;
+                const chart = this.series.chart;
```

Comment:
We are trying to use syntax to declare multiple variables after commas.
```suggestion
                const row = this.options.row;
                      chart = this.series.chart;
```

Reply:
fixed